server/build.gradle

```diff
@@ -14,7 +14,8 @@ def versions = [
     java: '11',
     flyway: '6.3.3',
     guava: '28.2-jre',
-    junit: '5.6.2'
+    junit: '5.6.2',
+    spdx: '2.1.20'
```

Comment:
The latest version is `2.2.1`.

https://search.maven.org/artifact/org.spdx/spdx-tools/2.2.1/jar